packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,278 @@
 :::
 
-#### Peer liquidity example
+If a peer’s liquidity is insufficient (e.g., they’ve used up their allotted credit line), payments will not be processed. Your peer should settle with you so that you can reset their liquidity.
+
+You can add a liquidity threshold for each peer liquidity account via the [`updatePeer`](https://rafiki.dev/apis/graphql/backend/mutations/#updatepeer) mutation's `liquidityThreshold` input argument.
+
+When a threshold is entered, the [`peer.liquidity_low`](/integration/requirements/webhook-events#peer-liquidity-low) webhook event will notify you if a peer's liquidity drops below the threshold.
+
+You should define and adjust each peer's liquidity based on your liquidity risk profile. You can deposit or withdraw peer liquidity as needed through [Rafiki Admin](/admin/admin-user-guide#edit-peer) or by using the [Backend Admin API](/admin/manage-liquidity#peer-liquidity).
+
+<details>
+<summary>Peer liquidity example</summary>
+You and Cloud Nine Wallet are peers. You’ve agreed to extend Cloud Nine Wallet a line of credit worth 100.00 USD. This means Cloud Nine Wallet has 100.00 in their peer liquidity account on your Rafiki instance.
+
+Your Rafiki instance can send packets that total up to 100.00 to wallet addresses issued by Cloud Nine Wallet. When the 100.00 is used up, Cloud Nine Wallet settles with you by sending 100.00 via the shared settlement mechanism outlined in your peering agreement. When you receive the funds, you reset their liquidity in Rafiki.
+
+</details>
+
+#### Payment liquidity accounts
+
+Payment liquidity is the amount that's available because of an incoming or outgoing payment. Rafiki has three types of payment liquidity accounts.
+
+<div class="overflow-table" style="border-bottom: 1px solid var(--sl-color-gray-6);">
+
+| Payment type                                         | Purpose                                                    |
+| ---------------------------------------------------- | ---------------------------------------------------------- |
+| [Incoming](#incoming-payment-liquidity-accounts)     | For incoming payments created via the Open Payments APIs   |
+| [Outgoing](#outgoing-payment-liquidity-accounts)     | For outgoing payments created via the Open Payments APIs   |
+| [Wallet address](#wallet-address-liquidity-accounts) | For incoming payments created via SPSP or Web Monetization |
+
+</div>
+
+##### Incoming payment liquidity accounts
+
+An incoming payment liquidity account represents the value received from a completed incoming payment. When an incoming payment is created via the <LinkOut href="https://openpayments.dev/apis/resource-server/operations/create-incoming-payment/">Open Payments APIs</LinkOut>, a corresponding liquidity account is automatically created. You will have one liquidity account per incoming payment.
```

Comment:
The account creation part actually happens when the first packet for the incoming payment is received.